### PR TITLE
Create wallabag v2 documentation - FR

### PR DESCRIPTION
**Pull request by lapineige**

### CODE REVIEW

File: app_wallabag2_fr.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## <img src="https://wallabag.org/user/themes/boxify/img/logo-wallabag-black.svg">
```
Comment (M5oul) [inline app_wallabag2_fr.md]: Do not gather images from other places.
You cloud [take this one](https://yunohost.org/images/Logo-wallabag-svg.svg)
You could also add a new image on `images` folder.

**Comment by M5oul** [comment]
Could you add a link towards this two pages on indexers pages.

File: app_wallabag2_fr.md (hunk 1 above)
Comment (lapineige) [inline app_wallabag2_fr.md]: I will use the one you mentioned, no need to duplicate the space for almost the same logo.
(I corrected the English version too)

**Comment by lapineige** [comment]
I'm not sure that I understand what you mean: on the apps.md files ?

**Comment by M5oul** [comment]
In [theses pages](https://yunohost.org/#/docs_fr).

**Comment by lapineige** [comment]
Ok, but which file should I change ?

**Comment by M5oul** [comment]
- https://yunohost.org/#/sitemap_fr
- https://yunohost.org/#/userdoc_fr

Also in english.

**Comment by lapineige** [comment]
Done. I added the mention to Wallabag's version (to avoid any confusion with the v1)

File: app_wallabag2_fr.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+## <img src="https://yunohost.org/images/Logo-wallabag-svg.svg">
+
+
+[Wallabag](https://www.wallabag.org/) est une application de lecture différée : elle  permet simplement d’archiver une page web en ne conservant que le contenu. Les éléments superflus (menus, publicités, etc.) sont supprimés.
+
+Sont disponibles: une interface web, des add-ons pour navigateurs (Firefox / Chrome / Opera), des applications pour mobile (Android / iOS / Windows Phone) et même sur liseuse (PocketBook / Kobo).
+
+avec YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=wallabag2)
```
Comment (M5oul) [inline app_wallabag2_fr.md]: Ça m’a l’air cassé ici.

Comment (lapineige) [inline app_wallabag2_fr.md]: Décidément, faut que je me relise, ça fait beaucoup de mauvaises manip' :/

**Pull request merged**